tmol/optimization/modules.py

```diff
@@ -74,6 +75,7 @@ def torsional_energy_network_from_system(
     )
 
 
+"""
```

Comment:
Commented this out with the above, not sure if technically necessary since I don't think it would have been parsed unless used.